packages/password/__tests__/accounts-password.ts

```diff
@@ -397,10 +431,12 @@ describe('AccountsPassword', () => {
       const userId = 'id';
       const setPassword = jest.fn(() => Promise.resolve('user'));
       const findUserById = jest.fn(() => Promise.resolve(validUser));
-      password.setStore({ setPassword, findUserById } as any);
+      const removeAllPasswordResetTokens = jest.fn(() => Promise.resolve());
```

Comment:
in this test, where do you check that it was actually called?

Reply:
My mistake. I checked the wrong method in the assertion.